app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_display_set_pdf_report.html

```diff
@@ -47,7 +49,7 @@ <h2 style="margin-top:50px;">Answers</h2>
                 {% for question, answer in answers.items %}
                     <tr>
                         <td class="left"><b>{{ question }}:</b></td>
-                        <td>{{ answer }}</td>
+                        <td>{{ answer|safe }}</td>

```

Comment:
It is always best to avoid the use of `safe`. Why is this needed here? Could you change the implementation to ensure this doesn't need to be used (either in a template filter or by calling it directly from python).